test/loki_service.cc

```diff
@@ -380,15 +445,10 @@ void start_service() {
                                              config.get<std::string>("httpd.service.interrupt"))));
   server.detach();
 
-  // load balancer
-  std::thread proxy(std::bind(&proxy_t::forward,
-                              proxy_t(context, config.get<std::string>("loki.service.proxy") + "_in",
-                                      config.get<std::string>("loki.service.proxy") + "_out")));
-  proxy.detach();
-
-  // service worker
-  std::thread worker(valhalla::loki::run_service, config);
-  worker.detach();
+  // proxies and workers
+  STAGE(loki);
+  STAGE(thor);
+  STAGE(odin);
```

Comment:
since the status request goes all the way through the service instead of just loki we have to actually run the other stages